shared/reducers/chat2.tsx

```diff
@@ -76,7 +76,9 @@ const botActions: Container.ActionHandler<Actions, Types.State> = {
     draftState.featuredBotsLoaded = loaded
   },
   [BotsGen.setSearchFeaturedAndUsersResults]: (draftState, action) => {
-    draftState.botSearchResults = action.payload.results
+    if (action.payload.results) {
```

Comment:
do we need this check?

Reply:
hmm, i guess not, could just have a reasonable fallback value instead